Cargo.toml

```diff
@@ -22,15 +22,15 @@ base-factory        = { version = "2.1.0", path = "contracts/factories/base-fact
 base-minter         = { version = "2.1.0", path = "contracts/minters/base-minter" }
 cosmwasm-schema     = "1.2.1"
 cosmwasm-std        = "1.2.1"
-cw-controllers      = "0.16.0"
-cw2                 = "0.16.0"
-cw4                 = "0.16.0"
-cw4-group           = "0.16.0"
-cw721               = "0.16.0"
-cw721-base          = "0.16.0"
-cw-multi-test       = "0.16.0"
-cw-storage-plus     = "0.16.0"
-cw-utils            = "0.16.0"
+cw-controllers      = "1.0.1"
+cw2                 = { git = "https://github.com/CosmWasm/cw-plus/", branch="main" }
```

Comment:
Why isn't this also 1.0.1?

Reply:
See the logic from cw-nfts, they wanted to use a non-main commit in order to get at the assert_contract_version function (note "main" is not part of the 1.0.1 yet):  https://github.com/CosmWasm/cw-nfts/pull/112/files#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542R18

Reply:
We need Confio to make a new release of cw2 that includes `assert_contract_version`